js/shared/Semgrep_js_shared.ml

```diff
@@ -42,11 +42,40 @@ let init_jsoo yaml_wasm_module =
      old: Parsing_init.init ();
   *)
   Yaml_ctypes_overrides.apply ();
+  Http_helpers.client_ref := Some (module Cohttp_lwt_jsoo.Client);
   Libyaml_stubs_js.set_libyaml_wasm_module yaml_wasm_module
 
 (*****************************************************************************)
 (* Entrypoints *)
 (*****************************************************************************)
+let setParsePattern (func : jbool -> jstring -> jstring -> 'a) =
```

Comment:
Please add a Semgrep_js_shared.mli with a small comment at the top explaining the goal of the module.

Reply:
will do